### PR TITLE
Adding deadline to Templates for blog posts

### PR DESCRIPTION
**Pull request by jakub-pomykala**

### CODE REVIEW

**Review by samwatibm — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/blog_post_beta.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ assignees: austin0, jakub-pomykala
 
 ---
 
-The information you provide here will be included in the Open Liberty beta blog post ([example](https://openliberty.io/blog/2020/08/05/jakarta-grpc-beta-20009.html)), which will be published on [openliberty.io/blog/](https://www.openliberty.io/blog/), and potentially elsewhere, to promote this beta feature/function of Open Liberty.
+The information you provide here will be included in the Open Liberty beta blog post ([example](https://openliberty.io/blog/2020/08/05/jakarta-grpc-beta-20009.html)), which will be published on [openliberty.io/blog/](https://www.openliberty.io/blog/), and potentially elsewhere, to promote this beta feature/function of Open Liberty. For this post to be included in the beta issue please make sure that this is completed by the end of Friday following the GM release (Wednesday).
```
Comment (tevans78) [inline .github/ISSUE_TEMPLATE/blog_post_beta.md:10]: suggest removing the word "release"

File: .github/ISSUE_TEMPLATE/blog_post_ga_release.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ assignees: austin0, jakub-pomykala
 
 ---
 
-The information you provide here will be included in the Open Liberty GA release blog post ([example](https://openliberty.io/blog/2020/08/05/jakarta-grpc-beta-20009.html)), which will be published on [openliberty.io/blog/](https://www.openliberty.io/blog/), and potentially elsewhere, to promote this newly released feature/function of Open Liberty.
+The information you provide here will be included in the Open Liberty GA release blog post ([example](https://openliberty.io/blog/2020/08/05/jakarta-grpc-beta-20009.html)), which will be published on [openliberty.io/blog/](https://www.openliberty.io/blog/), and potentially elsewhere, to promote this newly released feature/function of Open Liberty. For this post to be included in the GA issue please make sure that this is completed by the end of Friday following the GM release (Wednesday).
```
Comment (tevans78) [inline .github/ISSUE_TEMPLATE/blog_post_ga_release.md:10]: suggest removing the word "release"

File: .github/ISSUE_TEMPLATE/blog_post_beta.md (hunk 1 above)
Comment (jakub-pomykala) [reply .github/ISSUE_TEMPLATE/blog_post_beta.md]: Done

File: .github/ISSUE_TEMPLATE/blog_post_ga_release.md (hunk 2 above)
Comment (jakub-pomykala) [reply .github/ISSUE_TEMPLATE/blog_post_ga_release.md]: Done

**Review by tevans78 — APPROVED** [review]

**Comment by tevans78** [comment]
#run-libby-bot

**Pull request merged**